.firebaserc

```diff
@@ -28,21 +31,27 @@
         ],
         "api": [
           "reach4help-staging-api"
+        ],
+        "site": [
+          "reach4help-staging"
         ]
       }
     },
     "reach4help": {
       "hosting": {
         "web-client": [
-          "reach4help-web-client"
```

Comment:
When the project moved we lost this name to the old `reach4help-xxxx` project :(